model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -50,10 +50,24 @@ class InsertReverseChannels(BackReplacementPattern):
     """
     enabled = False
 
+    @staticmethod
+    def get_index_for_api_2_0(node: Node, idx: int):
+        if node.has_valid('rt_info'):
+            rt_info = node.rt_info
+            if rt_info.contains('old_api_map'):
+                old_api_map_version = rt_info.get_attribute_version('old_api_map')
+                old_api_map = rt_info.info['old_api_map', old_api_map_version]
+                if 'inverse_order' in old_api_map.info:
+                    order = old_api_map.info['inverse_order']
+                    assert len(order) > idx >= 0, 'Channel index is incompatible with old_api_map: {}.'.format(idx)
+                    return list(order).index(idx)
+        return idx
```

Comment:
Is it ok if node doesn't have rt_info/old_api_map? Should we output warning in this case?

Reply:
Yes, it's okay if node doesn't have old_api_map. This means that in new API layout and type is the same as in old API. Warning is not needed.